src/renderer/dx/DxRenderer.cpp

```diff
@@ -334,18 +394,41 @@ HRESULT DxEngine::_SetupTerminalEffects()
 
 // Routine Description:
 // - Puts the correct values in _pixelShaderSettings, so the struct can be
-//   passed the GPU.
+//   passed the GPU and updates the GPU resource.
 // Arguments:
 // - <none>
 // Return Value:
 // - <none>
 void DxEngine::_ComputePixelShaderSettings() noexcept
 {
-    // Retro scan lines alternate every pixel row at 100% scaling.
-    _pixelShaderSettings.ScaledScanLinePeriod = _scale * 1.0f;
+    if (_pixelShaderEffect && _d3dDeviceContext && _pixelShaderSettingsBuffer)
+    {
+        try

```

Comment:
Semantics changed so that this function now also updates the resource. This is because this is needed to be updated in more than one place now.